README-Finnish.md

```diff
@@ -0,0 +1,150 @@
+| englanti (English)                              | en    | -                                                                      | Käyttöliittymä ja README                        |
+| suomi                                           | fi     | [ZapX5](https://github.com/ZapX5)                                     | Käyttöliittymä ja [README](./README-Finnish.md) |
+| ranska (Français)                               | fr    | [flaviedesp](https://github.com/flaviedesp)                            | Käyttöliittymä                                  |
+| saksa (Deutsch)                                 | de    | [hebens](https://github.com/hebens), [ackhh](https://github.com/ackhh) | Käyttöliittymä                                  |
+| kreikka (Ελληνικά)                              | gr    | [sofronas](https://github.com/sofronas)                                | Käyttöliittymä                                  |
+| indonesia (Bahasa Indonesia)                    | id    | [MarvinZhong](https://github.com/MarvinZhong)                          | Käyttöliittymä                                  |
+| italia (Italiano)                               | it    | [itsEmax64](https://github.com/itsEmax64)                              | Käyttöliittymä                                  |
+| japani (日本語)                                  | ja    | [NattyanTV](https://github.com/nattyan-tv)                             | Käyttöliittymä                                  |
+| venäjä (Русский)                                | ru    | Oleg                                                                   | Käyttöliittymä                                  |
+| espanja (Español)                               | sp    | [enriiquee](https://github.com/enriiquee)                              | Käyttöliittymä                                  |
+| espanja (Latinalainen Amerikka) (Español Latam) | sp_la | [Matyrela](https://github.com/Matyrela)                                | Käyttöliittymä                                  |
+| serbia                                          | sr    | [rina](https://github.com/sweatshirts)                                 | Käyttöliittymä                                  |
+| thai (ภาษาไทย)                                 | th    | [teerut26](https://github.com/teerut26)                               | Käyttöliittymä (osittain)                        |
+| turkki (Türkçe)                                 | tr     | [mcagriaksoy](https://github.com/mcagriaksoy)                         | Käyttöliittymä                                  |
+| ukraina (Українська)                            | ua     | [AndrejGorodnij](https://github.com/AndrejGorodnij)                   | Käyttöliittymä                                  |
+
+> Haluatko lisätä uuden kielen? Päivitä [i18n.js-tiedostoa](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/auto_py_to_exe/web/js/i18n.js) ja lähetä Pull Request tai liitä se "issue":en.
```

Comment:
You can leave this part out - I just have it on the main README.md as it shows by default when visiting the repo.

We'll forget to add to this take so it might be easier to keep it out.